### PR TITLE
Add EventHub templates.

### PR DESCRIPTION
**Pull request by MikeStall**

### CODE REVIEW

File: Bindings/bindings.json [hunk 1]
```diff
@@ -7,23 +7,67 @@
         "triggerRefHelp": "You can use brackets ({value}) and percent symbols (%value%) to format your path based on parameters from the trigger object."
     },
     "bindings": [
-        {
-            "type": "timerTrigger",
-            "displayName": "Timer",
-            "direction": "trigger",
+      {
```
Comment (davidebbo) [inline Bindings/bindings.json]: Spacing is inconsistent with rest of file. I'm fine changing to 2 spaces, but then let's convert whole file

Comment (MikeStall) [inline Bindings/bindings.json]: I'm just using the VS defaults.

Comment (davidebbo) [inline Bindings/bindings.json]: Strange, VS default was 4 for me. But in any case, let's not fix in same file :)

**Comment by davidebbo** [comment]
:shipit: Just the spacing comment. That can be done separately

**Pull request merged**